main.py

```diff
@@ -3,14 +3,32 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    clock = pygame.time.Clock()
+    while True:
+        clock.tick(60) 
+        # Event handling to check if the user wants to quit
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                pygame.quit()
+                return
+            elif event.type == pygame.KEYDOWN:
```

Comment:
**Suggestion:** Implement handling for player key events to enhance interactivity.

**Existing Code:**
```Python
elif event.type == pygame.KEYDOWN:
```

**Improved Code:**
```Python
elif event.type == pygame.KEYDOWN:
    if event.key == pygame.K_LEFT:
        # Move player left
    elif event.key == pygame.K_RIGHT:
        # Move player right
```

**Details:**
Implement handling for player key events to enhance interactivity.